extensions/reviewed/InAppPurchase.json

```
@@ -8,7 +8,7 @@
   "name": "InAppPurchase",
   "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/Glyphster Pack/Master/SVG/Shopping and Ecommerce/Shopping and Ecommerce_wallet_money_cash.svg",
   "shortDescription": "Add items to buy directly in your game (\"In-App Purchase\"), for games published on Android or iOS.",
-  "version": "0.0.1",
+  "version": "1.0.0",
```

No need for a major bump here, also the extension quality is pretty poor still as far as I remember, and a 0.x.x release indicate it is not fully production ready, so I would keep it to something like this instead:
```suggestion
  "version": "0.0.2",
```